services/QuillLMS/db/migrate/20220609175032_change_sales_form_submission_columns.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class ChangeSalesFormSubmissionColumns < ActiveRecord::Migration[5.1]
+  def change
+    change_column_null :sales_form_submissions, :phone_number, true
+    change_column_null :sales_form_submissions, :zipcode, true
+    change_column_null :sales_form_submissions, :school_name, true
+    change_column_null :sales_form_submissions, :district_name, true
```

Comment:
school_name and district_name don't have the non-null constraint, so lines 7,8 should be removed because they are redundant.